tests/new_character_test.cpp

```diff
@@ -119,17 +106,18 @@ TEST_CASE( "starting_items" ) {
     };
     std::vector<failure> failures;
 
-    auto add_failure = [&]( const profession &prof, const std::vector<trait_id> &traits,
-                            const std::string &item_name, const std::string &reason ) {
-        if( !std::any_of( failures.begin(), failures.end(), [&item_name]( const failure &f ) {
+    auto add_failure = [&]( const profession & prof, const std::vector<trait_id> &traits,
+                            const std::string & item_name, const std::string & reason ) {
+        if( !std::any_of( failures.begin(), failures.end(), [&item_name]( const failure & f ) {
             return f.item_name == item_name;
-            } ) ) {
+        } ) ) {
```

Comment:
Multi-line lambdas in conditions still look terrible. Would be better if use a temp variable.